tests/ExampleSpec.hs

```diff
@@ -24,3 +25,11 @@ spec = do
         True `shouldBe` True
       it "gets skipped" $ do
         pendingWith "some reason"
+      it "throws a colourful exception" $ do
+        throwIO ColourfulException :: IO ()
+
+data ColourfulException = ColourfulException
+  deriving Show
```

Comment:
```suggestion
  deriving stock Show
```

^ This should fix the error in the `stack.yaml` build. Lack of a strategy has become an error-level warning as of that GHC version, it seems.